vue-thacer/src/assets/js/thacer-map-setup-search.js

```diff
@@ -28,7 +28,7 @@ export function setupSearchCeramByText(markerClusterGroupCeram, map) {
 
           // Adding current ceramObject only if unlocalised and passing input search string :
           if (
-            obj.properties.x != '' &&
+            obj.geometry.coordinates[0] === 0 &&
```

Comment:
coordinates (and geometry) properties always present ?
(sorry, I write that in every single PR haha)

Reply:
yes, because it is geojson and mandatory